src/Core/TileMesh.js

```diff
@@ -25,11 +25,10 @@ function TileMesh(geometry, params) {
     this.extent = params.extent;
 
     this.geometry = geometry;
-    this.normal = params.center.clone().normalize();
 
-    this.boundingSphereOffset = new THREE.Vector3();
+    this.obb = this.geometry.OBB.clone();
```

Comment:
What's the point of `this.obb`?

Reply:
obliged to clone because the DEM is different for each tile,